core/src/main/java/com/exadel/etoolbox/backpack/core/services/pckg/UploadPackageService.java

```diff
@@ -24,10 +23,10 @@ public interface UploadPackageService {
     /**
      * Method upload package by byte[] array representation
```

Comment:
(cosmetic) Should sound smth like `Uploads a content package as an array of bytes`. Does not have much meaning in the present form)